### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by faridaf97** [quasarframework/quasar#13426] [Quasar App Extensions] [answered]
Hi everyone ☺️

I'm new to App development world and need a kick start into installing Quasar, since i'm having trouble installing in onto my mac. I have installed node.js 16.15.01 LTS. When I tried installing Quasar afterwards, I've only received errors. What am I doing wrong? 

Last login: Sat May 14 14:34:51 on ttys000
faridafares@Faridas-MBP ~ % $ npm install -g @quasar/cli
zsh: command not found: $
faridafares@Faridas-MBP ~ % npm install -g @quasar/cli
npm notice 
npm notice New minor version of npm available! 8.5.5 -> 8.10.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.10.0
npm notice Run npm install -g npm@8.10.0 to update!
npm notice 
npm ERR! code EACCES
npm ERR! syscall mkdir
npm ERR! path /usr/local/lib/node_modules/@quasar
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/@quasar'
npm ERR!  [Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/@quasar'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'mkdir',
npm ERR!   path: '/usr/local/lib/node_modules/@quasar'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/faridafares/.npm/_logs/2022-05-14T12_50_51_230Z-debug-0.log
faridafares@Faridas-MBP ~ % npm install -g npm@8.10.0
npm ERR! code EACCES
npm ERR! syscall rename
npm ERR! path /usr/local/lib/node_modules/npm
npm ERR! dest /usr/local/lib/node_modules/.npm-i9nnxROI
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, rename '/usr/local/lib/node_modules/npm' -> '/usr/local/lib/node_modules/.npm-i9nnxROI'
npm ERR!  [Error: EACCES: permission denied, rename '/usr/local/lib/node_modules/npm' -> '/usr/local/lib/node_modules/.npm-i9nnxROI'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'rename',
npm ERR!   path: '/usr/local/lib/node_modules/npm',
npm ERR!   dest: '/usr/local/lib/node_modules/.npm-i9nnxROI'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/faridafares/.npm/_logs/2022-05-14T12_52_15_768Z-debug-0.log
faridafares@Faridas-MBP ~ %

**Comment by MilosPaunovic** [comment]
You don't need the `$` character. Just type `npm i -g @quasar/cli`.

**Reply by faridaf97** [reply]
Hi ☺️ 
Thank you for your reply! I've tried it out, but I still got an error :( 

Last login: Sat May 14 14:34:51 on ttys000
faridafares@Faridas-MBP ~ % npm i -g @quasar/cli
npm ERR! code EACCES
npm ERR! syscall mkdir
npm ERR! path /usr/local/lib/node_modules/@quasar
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/@quasar'
npm ERR!  [Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/@quasar'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'mkdir',
npm ERR!   path: '/usr/local/lib/node_modules/@quasar'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/faridafares/.npm/_logs/2022-05-14T12_58_34_804Z-debug-0.log
faridafares@Faridas-MBP ~ %

**Comment by MilosPaunovic** [comment, marked as answer]
You gotta have admin privilegies to install global packages.
Try `sudo npm i -g @quasar/cli`

**Reply by faridaf97** [reply]
I guess this worked! Thank you very much!!
